common/js/src/logging/debug-dump.js

```diff
@@ -121,14 +121,29 @@ function dumpNumber(value) {
  * @param {number} value
  * @return {string}
  */
-function dumpArrayBufferByte(value) {
+function dumpByte(value) {
   const HEX_LENGTH_OF_BYTE = 2;
   const hexValue = value.toString(16).toUpperCase();
   const zeroPadding =
       goog.string.repeat('0', HEX_LENGTH_OF_BYTE - hexValue.length);
   return zeroPadding + hexValue;
 }
 
+/**
+ * @param {!Uint8Array} value
+ * @return {string}
+ */
+function dumpBytes(value) {
```

Comment:
nit: dumpArrayByte to avoid confusion between "dumpByte" and "dumpBytes"

Reply:
That's reasonable, but maybe "dumpByteArray"?

Reply:
dumpByteArray sounds good